frontend/app/helpers/path-helper.js

```diff
@@ -76,6 +76,12 @@ module.exports = function() {
     queryPath: function(queryIdentifier) {
       return '/queries/' + queryIdentifier;
     },
+    repositoryPath: function(projectIdentifier) {
+      return PathHelper.projectPath(projectIdentifier) + '/repository';
+    },
+    revisionPath: function(projectIdentifier, revisionIdentifier) {
+      return PathHelper.repositoryPath(projectIdentifier) + '/revision/' + revisionIdentifier;
+    },
```

Comment:
ot to bitch here, but shouldn't we use HAL here and infer the links from the api response for the project?